pandas/io/tests/test_parsers.py

```diff
@@ -1036,7 +1036,7 @@ def test_parse_dates_string(self):
 20090103,c,4,5
 """
         rs = self.read_csv(
```

Comment:
can you move all of the parse_dates testing to a separate class (make it a base class, kind of like ``ParserTests`` e.g. it inherits from ``object`` and have ``ParserTests`` inherit from it. Ideally we would group tests like this to keep them together. Can do this in another PR if you want to organize things even more.